packages/p2p/src/hooks/useMarketRate.ts

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { useFetch } from '@deriv/api';
+import { useStore } from '@deriv/stores';
+import { useExchangeRate } from '@deriv/hooks';
+
+const useMarketRate = () => {
+    const { client } = useStore();
+    const { is_authorize } = client;
+    const { getRate } = useExchangeRate();
+    const { data } = useFetch('website_status', { options: { enabled: is_authorize } });
+    const override_exchange_rate = data?.website_status?.p2p_config?.override_exchange_rate;
+
+    const getMarketRate = React.useCallback(
+        (currency: string) => override_exchange_rate || getRate(currency),
+        [getRate, override_exchange_rate]
+    );
```

Comment:
Could you explain how this method operates? 🤔 I find it a bit unusual that we input a currency to obtain its rate relative to USD, but the `override_exchange_rate` parameter is only a string. What currency does it pertain to? 🤔

Reply:
I talked with bala for this, he told us base on our current structure we don't need to pass currency to this!

The way it works:
- They create an account for example ID country (local currency)
- They go to the BO and under the ID country they can change the rate
- BE will send the rate

so for now we don't need to implement this we just need to return it